src/lxc/lsm/selinux.c

```diff
@@ -76,12 +76,12 @@ static int selinux_process_label_set(const char *inlabel, struct lxc_conf *conf,
 {
 	const char *label = inlabel ? inlabel : conf->lsm_se_context;
 	if (!label) {
-		if (use_default)
-			label = DEFAULT_LABEL;
-		else
+		if (!use_default)
 			return -1;
+		label = DEFAULT_LABEL;
 	}
-	if (!strcmp(label, "unconfined_t"))
+
+	if (!strcmp(label, DEFAULT_LABEL))
```

Comment:
You shouldn't use `DEFAULT_LABEL` here. If we ever change `DEFAULT_LABEL` to something other than `unconfined_t` we'd want the code after this check to be executed. I'm taking your patch and adapt it and then push it to master. No need to rework it for you.

Reply:
OK, thanks!